lib/AST/ASTContext.cpp

```diff
@@ -715,6 +714,10 @@ ASTContext::~ASTContext() {
   getImpl().~Implementation();
 }
 
+void ASTContext::SetPreModuleImportCallback(std::function<void(llvm::StringRef ModuleName, bool IsOverlay)> callback) {
```

Comment:
Was this line `clang-format`ted? Looks pretty long

Reply:
It should've been, but I can also shorten it myself if `clang-format` is ignoring it for some reason

Reply:
If `clang-format` says it's ok then it's fine